frontends/ol-ckeditor/src/components/CkeditorArticle.tsx

```diff
@@ -54,7 +53,7 @@ const baseEditorConfig: EditorConfig = {
     ParagraphButtonUI,
     BlockEditorIcon,
   ],
-  blockToolbar: ["mediaEmbed", "imageUpload"],
+  // blockToolbar: ["mediaEmbed", "imageUpload"],
```

Comment:
I'm disabling this because they don't work well yet. See #123 and #124 for more information.